Kinvey.Core/Auth/User.cs

```diff
@@ -543,22 +569,50 @@ static public void LoginWithAuthorizationCodeLoginPage(string redirectURI, Kinve
 		/// <param name="password">Password for authentication</param>
 		/// <param name="redirectURI">The redirect URI to be used for parsing the grant code</param>
 		/// <param name="ct">[optional] CancellationToken used to cancel the request.</param>
+		[Obsolete("This method has been deprecated.  Please use LoginWithMIC() instead.")]
 		static public async Task LoginWithAuthorizationCodeAPIAsync(string username, string password, string redirectURI, AbstractClient userClient = null, CancellationToken ct = default(CancellationToken))
 		{
+			await LoginWithMIC(username, password, redirectURI, null, userClient, ct);
+		}
+
+		/// <summary>
+		/// Performs MIC Login authorization through an API.
+		/// </summary>
+		/// <param name="username">Username for authentication</param>
+		/// <param name="password">Password for authentication</param>
+		/// <param name="redirectURI">The redirect URI to be used for parsing the grant code</param>
+		/// <param name="ct">[optional] CancellationToken used to cancel the request.</param>
+		static public async Task LoginWithMIC(string username, string password, string redirectURI, string micID = null, AbstractClient userClient = null, CancellationToken ct = default(CancellationToken))
```

Comment:
@tejasranade The sync version of `LoginWithMIC` is for regular MIC flow, and the async version is for headless flow.  This is the way it has always been, but they used to be called `LoginWithAuthorizationCodeLoginPage` and `LoginWithAuthorizationCodeAPIAsync` respectively.  Since I was changing the parameter list of these methods, I took this opportunity to deprecate the old method names in favor of `LoginWithMIC`, which is overloaded.  Do you think I should name them slightly differently, to reflect their different uses?